dcos_launch/gce.py

```diff
@@ -0,0 +1,53 @@
+import json
+import logging
+
+from dcos_launch import util
+from dcos_test_utils import gce
+from dcos_test_utils.helpers import Host
+
+log = logging.getLogger(__name__)
+
+
+class BareClusterLauncher(util.AbstractLauncher):
+    # Launches a homogeneous cluster of plain GMIs intended for onprem DC/OS
+    def __init__(self, config):
+        credentials_path = util.set_from_env('GOOGLE_APPLICATION_CREDENTIALS')
+
+        with open(credentials_path, 'r') as creds:
```

Comment:
Nit: you can use https://github.com/mesosphere/dcos-test-utils/blob/master/dcos_launch/util.py#L36 to limit the number of places where we actually open files